Makefile

```diff
@@ -159,7 +159,7 @@ ci-lint:
 # dependency.
 prep:
 	@sh -c "'$(CURDIR)/scripts/goversioncheck.sh' '$(GO_VERSION_MIN)'"
-	@$(GO_CMD) generate $$($(GO_CMD) list ./... | grep -v /vendor/)
+	@GOARCH= GOOS= $(GO_CMD) generate $$($(GO_CMD) list ./... | grep -v /vendor/)
```

Comment:
I think you'll merge first but FYI I have a similar change in an Ent PR. I'll handle the conflict if you do though!